Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorGenerateSnapshot.cpp

```diff
@@ -142,9 +158,19 @@ bool SpatialGDKGenerateSnapshot(FString SavePath, UWorld* World)
 		}
 	}
 
-	const FString FullPath = FPaths::Combine(*SavePath, TEXT("default.snapshot"));
-	UE_LOG(LogSpatialGDKSnapshot, Display, TEXT("Saving snapshot to: %s"), *FullPath);
-	worker::SnapshotOutputStream OutputStream = worker::SnapshotOutputStream(improbable::unreal::Components{}, TCHAR_TO_UTF8(*FullPath));
+	SavePath = FPaths::Combine(*SavePath, SnapshotFileName);
+	return true;
+}
+
+bool SpatialGDKGenerateSnapshot(FString& SavePath, UWorld* World)
```

Comment:
Doesn't this need a definition in the header?

Reply:
It has one. See `SpatialGDKEditorGenerateSnapshot.h`.

Reply:
Oh yeah, got confused since there was a rename, but then SetupSnapshotGenerationPath isn't in the header, right?

Reply:
Yeah. It's just a helper for `SpatialGDKGenerateSnapshot`.

Reply:
Yeah, I have looked at the other implementations, and since this is done outside of classes, this makes sense - carry on